### PR TITLE
fork: use 8MB stack if rlimit returns unlimited 

### PR DESCRIPTION
**Pull request by MoZhonghua**
fix https://github.com/containers/youki/issues/213

### CODE REVIEW

File: src/process/fork.rs [hunk 1]
```diff
@@ -33,7 +35,13 @@ pub fn clone(cb: CloneCb, clone_flags: sched::CloneFlags) -> Result<Pid> {
         rlim_max: 0,
     };
     unsafe { Errno::result(libc::getrlimit(libc::RLIMIT_STACK, &mut rlimit))? };
-    let default_stack_size = rlimit.rlim_cur as usize;
+
+    // mmap will return ENOMEM if stack size is unlimited
+    let default_stack_size = if rlimit.rlim_cur != u64::MAX {
+        rlimit.rlim_cur as usize
+    } else {
+        DEFAULT_STACK_SIZE
```
Comment (utam0k) [inline src/process/fork.rs:43]: Could you have it output using `log::info!` with a reason to use the default stack size?

**Review by utam0k — COMMENTED** [review]
@MoZhonghua
Thanks for your first contribution.  I commented on this PR. Could you look at it?

File: src/process/fork.rs (hunk 1 above)
Comment (MoZhonghua) [reply src/process/fork.rs:44]: done

**Review by utam0k — APPROVED** [review]
LGTM

**Pull request merged**